src/modules/SX126x/SX126x.h

```diff
@@ -708,6 +708,16 @@ class SX126x: public PhysicalLayer {
     */
     int16_t setRxBandwidth(float rxBw);
 
+    /*!
+      \brief Enables or disables Rx Boosted Gain mode as described in SX126x datasheet section 9.6 (SX1261/2 v2.1, SX1268 v1.1)
+
+      \param rxbgm True for Rx Boosted Gain, false for Rx Power Saving Gain
+      \param retain True to persist gain setting through sleep/wake cycles
+
+      \returns 0
+    */
+    int16_t setRxBoostedGainMode(bool rxbgm = false, bool retain = false);
```

Comment:
I would suggest to remove the first default argument - it doesn't seem to make much sense to call `setRxBoostedGainMode()`, why is `false` the default?

Reply:
Ah yep agreed, defaulting this to false does not make sense – removed.